app/networkapi/highlights/migrations/0001_initial.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.6 on 2017-06-16 20:12
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import mezzanine.core.fields
+import networkapi.highlights.models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Highlight',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(help_text='Title of the higlight', max_length=300)),
+                ('description', models.TextField(help_text='Description of the higlight', max_length=5000)),
+                ('link_label', models.CharField(help_text="Text to show that links to this higlight's details page", max_length=300)),
+                ('link_url', models.URLField(help_text="Link to this higlight's details page", max_length=2048)),
+                ('image', models.FileField(help_text='Image representing this highlight', max_length=2048, upload_to=networkapi.highlights.models.get_highlights_image_upload_path)),
+                ('featured', models.BooleanField(default=False, help_text='Do you want to feature this highlight?')),
+                ('footer', mezzanine.core.fields.RichTextField(help_text='Content to appear after description (view more projects link or something similar)', null=True, verbose_name='footer')),
```

Comment:
This is a dubious name: "footers" are things at the bottom of a page, whereas this refers to (near as I can figure) something that allows the user to click through to "something". Can we change the name for this field?